src/main/java/io/github/thebusybiscuit/slimefun4/implementation/guide/SurvivalSlimefunGuide.java

```diff
@@ -57,7 +57,7 @@
  * The {@link SurvivalSlimefunGuide} is the standard version of our {@link SlimefunGuide}.
  * It uses an {@link Inventory} to display {@link SlimefunGuide} contents.
  *
- * @author TheBusyBiscuit
+ * @author Svdermant
```

Comment:
Add your name, dont change the original ones

Reply:
In you documentation stand i should add a @author tag that identified where modified the code... or so..

Reply:
add, not change

Reply:
oh sry than i missunderstand this i noted this for later coding if i must do this.. I'm a newbee :)

Reply:
> oh sry than i missunderstand this i noted this for later coding if i must do this.. I'm a newbee :)

You can note it for later but you need to change it properly in this PR before it can get merged 👍

Reply:
so must i write @author TheBusyBiscuit, svdermant? or with a space

Reply:
you add a new line with @author